modules/015-admission-policy-engine/hooks/bootstrap_handler.go

```diff
@@ -44,10 +54,23 @@ var _ = sdk.RegisterFunc(&go_hook.HookConfig{
 			},
 			FilterFunc: filterGatekeeperDeployment,
 		},
+		{
+			Name:       "gatekeeper_templates",
+			ApiVersion: "templates.gatekeeper.sh/v1",
+			Kind:       "ConstraintTemplate",
+			LabelSelector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"heritage": "deckhouse",
+					"module":   "admission-policy-engine",
+				},
+			},
+			FilterFunc: filterGatekeeperTemplates,
+		},
 	},
 }, handleGatekeeperBootstrap)
 
 func handleGatekeeperBootstrap(input *go_hook.HookInput) error {
+	var existingTemplates = make(map[string]struct{})
 	snap := input.Snapshots["gatekeeper_deployment"]
```

Comment:
We can remove deployment check. We don't care is it running or not, we just want to have present CRDs in a cluster